education.html

```diff
@@ -386,6 +388,26 @@ <h4 class="text-center pb-2 pt-5" data-aos="fade-up" data-aos-delay="300">MOOCs<
   <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
   <script src="https://unpkg.com/aos@2.3.1/dist/aos.js"></script>
   <script type="text/javascript" src="./assets/js/education.js"></script>
+  <!-- When the user clicks on the button, scroll to the top of the document -->
+  <script>
+    //Get the button
+var mybutton = document.getElementById("myBtn");
```

Comment:
The overall code structure and documentation are fantastic. Could you please adjust the indentation for it so that when we add the code quality checkers there is no issue with this.

For example:

```js
  <script>
       window.onscroll = function() {scrollFunction() };
       function scrollFunction( ){
             if (document.body.scrollTop >200 || document.documentElement.scrollTop > 200) {
                  mybutton.display = "block";
              }
             else{
                   mybutton.display = "none";   
              }
       .
       .
       .
   </script>
```